packages/synapse-react-client/test/lib/containers/download_list/DownloadDetails.test.tsx

```diff
@@ -2,74 +2,49 @@ import * as React from 'react'
 import DownloadDetails, {
   DownloadDetailsProps,
 } from '../../../../src/lib/containers/download_list/DownloadDetails'
-import ReactDOM from 'react-dom'
```

Comment:
Update how the component is rendered to resolve this warning --- `ReactDOM.render is no longer supported in React 18. Use createRoot instead.`